.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -0,0 +1,13 @@
+---
+name: Bug report
+about: Share about things that are not working as expected
+
+---
+
+/kind bug
+
+**Bug description:**
+[Describe the bug in detail including how to reproduce it. Please include any relevant output or screenshots.]
```

Comment:
breaking this section into something like the following would help
```
What happened:
What you expected to happen:
How to reproduce it (as minimally and precisely as possible):
```
e.g. https://github.com/kubernetes/kubernetes/blob/master/.github/ISSUE_TEMPLATE/bug-report.md

Reply:
Fixed.